jax/_src/scipy/special.py

```diff
@@ -1719,3 +1720,47 @@ def bernoulli(n: int) -> Array:
   k = jnp.arange(2, 50, dtype=bn.dtype)  # Choose 50 because 2 ** -50 < 1E-15
   q2 = jnp.sum(k[:, None] ** -m[None, :], axis=0)
   return bn.at[4::2].set(q1 * (1 + q2))
+
+
+@_wraps(osp_special.comb, module='scipy.special')
+def comb(N: Array, k: Array, exact: bool =False, repetition: bool =False):
+  r"""The number of combinations of N things taken k at a time.
+
+  This is often expressed as "N choose k".
+
+  Args:
+    N: The number of things.
+    k: The number of elements taken.
+    exact: If `True`, the result is computed exactly and returned as an integer type.
+      Currently, vectorization is not supported for exact=True.
+    repetition: If `repetition` is True, then the number of combinations with
+      repetition is computed.
+    
+  Returns:
+    The number of combinations of N things taken k at a time.
+
+  Notes:
+  When exact=False, the result is approximately and efficiently computed using the following formula:
+
+  .. math::
+    \begin{equation}
+    \exp\left\{\ln{\Gamma(N+1)} - [\ln{\Gamma(k+1)} + \ln{\Gamma(N+1-k)}]\right\}
+    \end{equation}
+  
+  Where we use the Gamma function. 
+  """
```

Comment:
You can remove this entire docstring, it will be overwritten by the `_wraps` decorator above.